README.md

```diff
@@ -83,12 +83,12 @@ Does your company use Mocha?  Ask your manager or marketing team if your company
 
 You might want to know that:
 
-- Mocha is the *most-depended-upon* module on npm (source: [libraries.io](https://libraries.io/search?order=desc&platforms=NPM&sort=dependents_count)), and
-- Mocha is an *independent* open-source project, maintained exclusively by volunteers.
+- Mocha is the _most-depended-upon_ module on npm (source: [libraries.io](https://libraries.io/search?order=desc&platforms=NPM&sort=dependents_count)), and
+- Mocha is an _independent_ open-source project, maintained exclusively by volunteers.
```

Comment:
Are these changed by markdown-linter automatically?

Reply:
Yes. I think markdown-linter automatically.. I will fix it. thank you